makefiles/build.mk

```diff
@@ -11,6 +13,7 @@ start-evm:
 
 rollup:
 	@. ${CURDIR}/evm/scripts/start-celestia-devnet.sh && \
+	trap 'docker compose down chain celestia-devnet' EXIT; \
```

Comment:
Running your test instructions didn't seem to work.

I see the error `unknown command "chain" for "docker compose down"`. I think the `docker-compose down` doesn't actually take the list of containers... which seems at odds with the documentation here: https://docs.docker.com/reference/cli/docker/compose/down/. 

I'm on `Docker Compose version v2.17.3`

I think you either want to 1) use `docker compose down` to stop all containers or 2) use `docker compose stop chain celestia-devnet`

Reply:
@jerargus Yes, I just found out that the feature to selectively 'compose down' a service was only introduced after Docker v2.19.x (https://github.com/docker/compose/pull/10552). Currently, I'm running v2.24.x on my machine. 

I think it's fine to just tear down all the services with `docker compose down` for broader compatibility with older Docker versions, it will have same behaviour since the container running on `make rollup` is only chain & celestia-devnet anyway. Also, I'm avoiding `docker compose stop` as it still leaves the container there in a "stopped" status.

Reply:
I've just push the update. Thanks for pointing it out!

Reply:
👍 I'll update docker on my end as well.